crypto/src/test/java/org/web3j/crypto/WalletUtilsTest.java

```diff
@@ -154,4 +166,12 @@ public void testIsValidAddress() {
         assertFalse(isValidAddress(SampleKeys.ADDRESS + 'a'));
         assertFalse(isValidAddress(SampleKeys.ADDRESS.substring(1)));
     }
+
+    private byte[] sha256(byte[] input) {
```

Comment:
How come this is defined again here?

Reply:
Moved to `Hash` utility class in `utils` module

Reply:
Looks like this code hasn't been removed yet - please remove, and I'll merge.